### PR TITLE
add baseurl functionality to inplace urls

### PR DESCRIPTION
**Pull request by ZumZoom**

### CODE REVIEW

**Comment by Alir3z4** [comment]
@ZumZoom :+1: for this.
I like it.

Could you please also follow https://github.com/Alir3z4/html2text/blob/master/docs/contributing.md so we can merge it right away ?

**Comment by ZumZoom** [comment]
Done.

Also patched the same thing with \ tags.

**Comment by Alir3z4** [comment]
Awesome, I'll be waiting to travis to finish it's job and I'll be merging right away.

File: test/test_html2text.py [hunk 1]
```diff
@@ -173,6 +173,12 @@ def test_func(self):
     if base_fn not in ['bodywidth_newline.html', 'abbr_tag.html']:
         test_func = None
 
+    if base_fn == 'inplace_baseurl_substitution.html':
+        module_args['baseurl'] = 'http://brettterpstra.com'
+        module_args['body_width'] = 0
+        # there is no way to specify baseurl in cli :(
```
Comment (Alir3z4) [inline test/test_html2text.py]: @ZumZoom Would be good to add such feature?

**Comment by Alir3z4** [comment]
@ZumZoom Merged the pull-request.
Thanks for the great contribution and code.

File: test/test_html2text.py (hunk 1 above)
Comment (ZumZoom) [inline test/test_html2text.py]: It depends on whether people use cli with saved html pages or not.
I use cli occasionally for testing purposes with http://* addresses only. So personally I don't need such feature.

**Pull request merged**